### ISSUE TITLE
A generic way to call a custom AMCP-command string

### ISSUE THREAD

**Issue by z1glr** [SuperFlyTV/casparcg-connection#204]
* **I'm submitting a:** feature request

* **Summary**
In a past version, there was a method to send any AMCP-command as a string (referenced at #122).
In the current library-verison (v6) I can't find any equivalent command.

**Comment by mint-dewit** [comment]
See https://github.com/SuperFlyTV/casparcg-connection/pull/205

**Comment by mint-dewit** [comment]
Could you have a go with version `6.3.0-nightly-master-20240412-065100-23103b26.0` and see how that works for you?

**Comment by z1glr** [comment]
I did some testing and so for it's working great. Some things I tested: (both in CasparCG-verison 2.3.3 and 2.4.0-RC1)

- Simple commands like `PLAY` / `PAUSE` / `RESUME` / `CLEAR`
- `CG ADD` together with JSON-data. The JSON has to be escaped manually which I think also should be this way, since it's exactly the CLI behaves. I didn't test XML-data, since I don't have any templates using it.

What doesn't work is the use of `REQ`: Adding a `REQ`-id to the command, Since the library uses `REQ` itself, send commands get ignored. I think it could either be let as is with maybe some documentation somewhere about this behaviour or the presence of a user-defined id could be detected and used instead to be behave more like the direct CLI
